index.bs

```diff
@@ -1299,7 +1299,7 @@ used to declare that all objects implementing an [=interface=] |I|
 (identified by the first [=identifier=])
 must additionally include the [=interface mixin member|members] of [=interface mixin=] |M|
 (identified by the second identifier).
-[=Interface=] |I| is said to <dfn>include</dfn> [=interface mixin=] |M|.
+[=Interface=] |I| is said to <dfn export>include</dfn> [=interface mixin=] |M|.
```

Comment:
I wonder if we need to namespace it with for=interface. But probably fine?

Reply:
(If we do that you want to add id=include to not change that.)

Reply:
What's the best practice here?

Reply:
Any established patterns don't really apply to IDL I think.

Reply:
Well, given we foresee Dictionary mixins, I guess namespacing here makes sense.